sources/packages/web/src/views/aest/student/AssessmentAward.vue

```diff
@@ -59,14 +66,19 @@ export default defineComponent({
     const assessmentAwardData = ref<AwardDetailsAPIOutDTO>();
     const { mapAssessmentDetailHeader } = useAssessment();
     const headerMap = ref<Record<string, string>>({});
+    const snackBar = useSnackBar();
 
     onMounted(async () => {
+      loadAssessmentAwardValues();
```

Comment:
`onMounted(loadAssessmentAwardValues)`

Reply:
`loadAssessmentAwardValues` is already awaiting, so I think the `async` can be removed as Ann mentioned.